fix: [PL-46557]: Enhance documents, add examples (#8)
harness/delegate-dockerfile: README.md

The two lines below will copy all the certificates from local ./my-custom-ca to /opt/harness-delegate/my-ca-bundle/ directory inside the container.

```yaml
USER root
  
RUN curl -s -L -o delegate.jar $BASEURL/$DELEGATEVERSION/delegate.jar

+ COPY <path to the directory of my local certs> <path to the directory of certs inside the container>

+ RUN bash -c "/opt/harness-delegate/load_certificates.sh <path to the directory of certs inside the container>"

USER 1001
```

> Note that 

Reviewer: ```suggestion
### Important notes
```

> 1. Please DO NOT copy your certificates into the folder at `/opt/harness-delegate/ca-bundle`. This is a reserved folder used for installing additional certificates a delegate starts.

Reviewer: ```suggestion
1. Please DO NOT copy your certificates into the folder at `/opt/harness-delegate/ca-bundle`. This is a reserved folder used for installing additional certificates when a delegate starts.
```

> 1. please DO NOT copy your certificates into folder at `/opt/harness-delegate/ca-bundle`, because this is a reserved folder used for installing additional certificates to at the time of starting a delegate.

Reviewer: ```suggestion
1. Please DO NOT copy your certificates into the folder at `/opt/harness-delegate/ca-bundle`. This is a reserved folder used for installing additional certificates a delegate starts.
```

> 2. Set user to root before running the load_certificates.sh script. Then set it back to normal user after.

Reviewer: ```suggestion
2. Set the user to root before running the `load_certificates.sh` script. Then the user back to normal access after you run the script.
```
